cli/create.go

```diff
@@ -85,12 +92,16 @@ var createCLICommand = cli.Command{
 // Use a variable to allow tests to modify its value
 var getKernelParamsFunc = getKernelParams
 
-func create(containerID, bundlePath, console, pidFilePath string, detach bool,
+func create(ctx context.Context, containerID, bundlePath, console, pidFilePath string, detach bool,
 	runtimeConfig oci.RuntimeConfig) error {
 	var err error
 
+	span, ctx := opentracing.StartSpanFromContext(ctx, "create")
```

Comment:
I'm curious how much overhead (time wise) this tracing will add ? Given the fact that we have a jeager agent running (as you said this would resolve into no-op otherwise).

Reply:
Good question. I've written a test program to create spans using a "real" tracer and a "NOP" tracer (as this branch now uses by default). Creating `100,000` spans on a NUC:

| | nanonseconds / span |
|----|---|
| **Real tracer** | `2860` |
| **NOP tracer** | `550` |

So, there is some overhead even with tracing "disabled" but when you consider that a `docker run -ti busybox true` with this branch currently only creates `17` spans, I'd say the overhead is unlikely to be an issue, particularly when comparing that with the value tracing brings.

Reply:
Thanks for providing the data, this looks negligible indeed.